Unable to run (or build) the application on Android

Hi, I have the following error and it is impossible for me to run the app on Android. On iOS (where I have tested so far) it is working like a charm. 

```
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconHelper.kt: (155, 24): Type mismatch: inferred type is String? but String was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':beacons_plugin:compileDebugKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 43s
The built failed likely due to AndroidX incompatibilities in a plugin. The tool is about to try using Jetfier to solve the incompatibility.
Building plugin beacons_plugin...
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/AppUtils.kt: (14, 17): Unresolved reference: core
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/AppUtils.kt: (66, 24): Unresolved reference: NotificationCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/AppUtils.kt: (73, 26): Unresolved reference: NotificationCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (11, 17): Unresolved reference: core
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (12, 17): Unresolved reference: core
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (168, 20): Unresolved reference: ContextCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (169, 21): Unresolved reference: ContextCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (200, 29): Unresolved reference: ActivityCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (204, 29): Unresolved reference: ActivityCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (253, 24): Unresolved reference: ContextCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (254, 25): Unresolved reference: ContextCompat
e: /Users/Home/Documents/SDKs/flutter/.pub-cache/hosted/pub.dartlang.org/beacons_plugin-1.0.19/android/src/main/kotlin/com/umair/beacons_plugin/BeaconsPlugin.kt: (263, 21): Unresolved reference: ContextCompat

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileReleaseKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 59s

Exception: The plugin beacons_plugin could not be built due to the issue above.
```

Any advice will be highly appreciated!

--- a/android/src/main/kotlin/com/umair/beacons_plugin/BeaconHelper.kt
+++ b/android/src/main/kotlin/com/umair/beacons_plugin/BeaconHelper.kt
@@ -152,7 +152,7 @@ open class BeaconHelper(var context: Context) : BeaconConsumer, BeaconsPlugin.Co
             beaconManager?.startRangingBeaconsInRegion(region)
         } catch (e: RemoteException) {
             e.printStackTrace()
-            Log.e(TAG, e.message)
+            Log.e(TAG, e.message.toString())
         }
     }
 

